### ISSUE TITLE
__func__

### ISSUE THREAD

**Issue by tenk-a** [LADSoft/OrangeC#35]
Using __func__ will cause errors and warnings.

smp.c | smp.cpp:
```
int main(void) {
    const char* p = __func__;
    return 0;
}
```

> occ /1 smp.c
Warning(132)  smp.c(2):  Suspicious pointer conversion in function main

> occ /1 smp.cpp
Error(204)    smp.cpp(2):  Cannot convert ' const char *' to ' const char *' in function main

**Comment by LADSoft** [comment]
fixed in 6.0.23

**Comment by tenk-a** [comment]
It was fixed at 6.0.23.1
Thank you
